### PR TITLE
Links do not render correctly on Windows #423

### PR DESCRIPTION
**Pull request by rahul3v**
The error has been resolved and tested by running locally ...!

### CODE REVIEW

File: src/layouts/articleLayout.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import SEO from '../components/seo'
 import { graphql } from 'gatsby'
 import MDXRenderer from 'gatsby-plugin-mdx/mdx-renderer'
 import { useNavigate } from '@reach/router'
+import { MDXProvider } from "@mdx-js/react"
```
Comment (nilubava) [inline src/layouts/articleLayout.tsx]: @rahul3v This is not the right place to define new MDXProvider components. It should be defined in `src/components/customMdx/index.tsx`
Also, this is a path issue which needs to be resolved on the plugin level, which you can find inside `plugins/` folder

**Review by nilubava — CHANGES_REQUESTED** [review]

File: src/layouts/articleLayout.tsx (hunk 1 above)
Comment (rahul3v) [reply src/layouts/articleLayout.tsx]: ok @nilubava, Well there are too many problems, even if we re fix the only file root path, the link still breaks , so i have to write code to run specifically for local run, see the next commit

**Comment by nilubava** [comment]
@rahul3v It works on Windows. Thanks!Great job!
Once you resolve the conflicts, we can merge it.

**Comment by rahul3v** [comment]
Well @nilubava, I have reviewed all the links, the updated commit was working fine ...!, the warning links will come just because of using some 'redirects' and avoiding '/index' link format.  So, it's not an issue,
you can merge the last commit

**Comment by rahul3v** [comment]
@nilubava  you can merge it now

**Pull request merged**